utils/metrics/create_confusion_matrix.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+
+def create_confusion_matrix(prediction: list, true_y: list, save_location: str):
+    """
+    This function makes confusion matrix and saves it 
+    """
+    # Create confusion matrix in pandas
+    results = pd.DataFrame({"prediction": prediction, "expected": true_y})
+    results["combined"] = results["prediction"] + "_" + results["expected"]
+    confusion_matrix = results.pivot_table(
+        values="combined",
+        index="prediction",
+        columns="expected",
+        aggfunc=lambda x: len(x),
+    ).fillna(0)
+
+    # Create plot
+    fig = plt.figure(figsize=(18, 16), dpi=80, edgecolor="k")
+    sns.heatmap(confusion_matrix, annot=True, fmt="g")
+
+    os.makedirs(save_location, exist_ok=True)
+    plt.savefig(os.path.join(save_location, "confusion_matrix.png"))
+
+    # save pandas confusion_matrix
+    confusion_matrix.to_excel(os.path.join(save_location, "confusion_matrix.xlsx"))
```

Comment:
curious to why we're saving to a `.xlsx` file? Won't `.csv` or `.tsv` work just as well for this specific file? (it's just a flat file right?)

A `.xlsx` file can't be [previewed on github](https://github.com/cytodata/single-cell-classifier/blob/0423c9726f810e8c8438811ca43ad76ff833ae71/3.train_ebimage/1.random_forest/results/5/confusion_matrix.xlsx) but `.csv` and `.tsv` [files can be previewed](https://github.com/cytodata/single-cell-classifier/blob/master/2.process-data/results/target_counts.tsv).

If you decide to move towards `.csv` and `.tsv` (they can be open easily by any file reader too) then we can remove [`xlrd` and `openpyxl`](https://github.com/cytodata/single-cell-classifier/pull/27/files#diff-b7cb75ca59993a879e29bed8cd8469b8R14-R15) from the `environment.yml` too.